src/components/Feeds/FeedPosts/FeedPosts.jsx

```diff
@@ -118,10 +118,10 @@ const FeedPosts = ({ feeds, searchTerm, showOnlyFollowingPosts, isFeedLoading, d
         //                                 key={index}
         //                                 user={user}
         //                                 feed={feed}
-        //                                 comments={feedCommentsData({ feedId: feed._id })}
-        //                                 likes={feedLikesData({ feedId: feed._id })}
-        //                                 bookmarks={feedUserBookmarksData({ feedId: feed._id })}
-        //                                 views={feedViewsData({ feedId: feed._id })}
+        //                                 comments={feedCommentsData({ feedId: feed?._id })}
+        //                                 likes={feedLikesData({ feedId: feed?._id })}
+        //                                 bookmarks={feedUserBookmarksData({ feedId: feed?._id })}
+        //                                 views={feedViewsData({ feedId: feed?._id })}
```

Comment:
Remove comments if not needed (I think that those changes are from your prettier)

Reply:
I think we need those comments are for implementing scroll in FeedPosts so I will leave them for now